lte/gateway/python/magma/mobilityd/dhcp_client.py

```diff
@@ -132,11 +132,7 @@ def send_dhcp_packet(
             pkt_xid = self._msg_xid
             ciaddr = dhcp_desc.ip
         else:
-            LOG.warning(
-                "Unknown egress request mac %s state %s",
-                str(mac),
-                state,
-            )
+            LOG.warning(f"Unknown egress request mac {str(mac)} state {state}")
```

Comment:
For logging we actually enforce the other syntax because it is lazy (see the failing linter test).